examples/jsm/renderers/common/nodes/NodeStorageBuffer.js

```diff
@@ -4,11 +4,13 @@ let _id = 0;
 
 class NodeStorageBuffer extends StorageBuffer {
 
-	constructor( nodeUniform ) {
-
-		super( 'StorageBuffer_' + _id ++, nodeUniform ? nodeUniform.value : null );
+	constructor( nodeUniform, readOnly) {
+    readOnly = readOnly ? readOnly : false;
+    const namePrefix = readOnly ? 'StorageReadOnlyBuffer_' : 'StorageBuffer_'
+		super( namePrefix + _id ++, nodeUniform ? nodeUniform.value : null );
 
 		this.nodeUniform = nodeUniform;
+    this.readOnly = readOnly;
```

Comment:
Please use tabs.

Reply:
Apologies for the delay, hopefully formatting should be fixed now that I've manually checked each file as well as disabled a lot of the automatic spacing and indentation VSCode was applying to the files.

Reply:
This should also be true for pull request #28379 as well.